reports/kellsonphilips/Second-Task-start/Fitness-Management-Record/morning-fitness-routine.md

```diff
@@ -92,3 +92,16 @@
 - Neck exercise 12 in 4x reps : 48      
 - Tricep exercise 20 in 5x reps : 100 
 <hr>
+
+## Date: 28/10/2022
+ - [x] Check if satified
```

Comment:
I think I have to fix this now.. change satisfied to "satisfied" haha

Reply:
haha thanks for pointing it out. I will change it in another PR please 😂